src/wrappers/checks.ts

```diff
@@ -0,0 +1,72 @@
+import {ChecksApi} from '../api'
+import {Check, ServiceOptions} from '../types'
+import {addLabelDefaults} from './labels'
+import {AxiosResponse} from 'axios'
+
+const addDefaults = (check: Check): Check => ({
+  ...check,
+  labels: (check.labels || []).map(addLabelDefaults),
```

Comment:
I agree that they ensuring certain parameters exist is not the concern of the client, but I'd rather remove them all at once once we decide to do that, instead of having an inconsistent pattern. Adding an issue to take a look at label defaults